sdk/messaging/azservicebus/ci.yml

```diff
@@ -0,0 +1,18 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/messaging/azservicebus
+
+pr:
+  paths:
+    include:
+    - sdk/messaging/azservicebus
+
+stages:
+- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: 'servicebus'
+    RunTests: true
+    RunLiveTests: true
+    
```

Comment:
nit: end of file

Reply:
Trimmed the extra whitespace on the last line but left the newline at end (not sure actually if that's what you're pointing out).

Reply:
Yep that was it.